serverless/prisma/client/index.ts

```diff
@@ -0,0 +1,32 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+async function main() {
```

Comment:
I don't know if this lib needs a "main" function.  We should create a function called `getAllTokens` or similar and put this logic there.

Reply:
maybe just leave a comment saying that is a example is enough